public/static/docs/get-started/pipeline.md

```diff
@@ -36,8 +36,8 @@ $ dvc push
 ```
 
 This example is simplified just to show you a basic pipeline, see a more
-advanced [example](/doc/tutorials/pipelines) or complete
-[tutorial](/doc/tutorials/deep) to create a
+advanced [example](/doc/tutorials/pipelines) or
+[complete tutorial](/doc/tutorials/pipelines) to create a
```

Comment:
the most complete is probably the deep one, though it the oldest and most outdated

Reply:
This link is to the "complete tutorial to create an NLP pipeline". That's `/doc/tutorials/pipelines`, right?